src/go/rdctl/cmd/root.go

```diff
@@ -77,16 +82,30 @@ func init() {
 	rootCmd.PersistentFlags().StringVar(&password, "password", "", "overrides the password setting in the config file")
 }
 
-func doRequest(method string, command string) ([]byte, error) {
+func versionCommand(version string, command string) string {
+	if version == "" {
+		return fmt.Sprintf("%s/%s", apiVersion, command)
+	}
+	return fmt.Sprintf("%s/%s", version, command)
+}
+
+func makeURL(host string, port string, command string) string {
+	if command[0] == '/' {
```

Comment:
Consider using [`strings.HasPrefix`](https://pkg.go.dev/strings#HasPrefix). In particular, it deals with empty `command` (which would panic, except that I believe the caller deals with it).

Reply:
Not needed for now, because the caller is now checking for and complaining about an endpoint of empty string, but whatever. If there's still a panic it'll be on me.